python/triqs_dft_tools/converters/plovasp/proj_shell.py

```diff
@@ -196,7 +189,7 @@ def extract_tmatrices(self, sh_pars):
 # If no transformation matrices are provided define a default one
         self.do_transform = False
 
-        ns_dim = 2 if self.nc_flag else 1
+        ns_dim = 1 #2 if self.nc_flag else 1
```

Comment:
is this correct? Can we remove the commented section if we want to keep it?

Reply:
You can remove the commented section, yes